<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Create Account</string>
  </property>
  
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>30</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QWidget" name="formContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_header">
        <property name="text">
         <string>Create Account</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_subtitle">
        <property name="text">
         <string>Welcome</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_username">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineedit_username">
        <property name="placeholderText">
         <string>Enter your username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_password">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineedit_password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Create a password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_confirm_password">
        <property name="text">
         <string>Confirm Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineedit_confirm_password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Re-enter your password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_register">
        <property name="minimumSize">
         <size>
          <width>298</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_login_prompt">
        <property name="text">
         <string>Already have an account?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_login">
        <property name="text">
         <string>Sign in instead</string>
        </property>
       </widget>
      </item>
     </layout>
     <widget class="QLabel" name="errorLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>16</width>
        <height>24</height>
       </rect>
      </property>
      <property name="visible">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="rightPanel" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_avatar">
        <property name="styleSheet">
         <string notr="true">font-size: 18px; font-weight: 600; color: #1e293b;</string>
        </property>
        <property name="text">
         <string>Profile Picture</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="avatarPreview">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">qproperty-pixmap: url(:/icons/default-avatar.svg);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_select_avatar">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Select Avatar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_avatar_hint">
        <property name="styleSheet">
         <string notr="true">color: #64748b; font-size: 12px;</string>
        </property>
        <property name="text">
         <string>JPG or PNG, max 2MB</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
